versioned_docs/version-0.x.0/kubeslice-cli/kubeslice-cli-commands/completion.mdx

```diff
@@ -0,0 +1,26 @@
+# Completion [work-in-progress]
```

Comment:
Can you remove it? This topic with `work in progress` may or may not be part of Google search results or website search results. It won't look good to have this `in progress` page on the production website.

Reply:
deleting the page, as it is not required.